hack/install-pipeline-service.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -e
+
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..
+
+source ${ROOT}/hack/flags.sh "The install-pipeline-service.sh installs Pipeline Service for development and testing on non-production clusters / kcp instances."
+MODE=${MODE:-preview} parse_flags $@
+
+
+PIPELINE_SERVICE_DIR=$(mktemp -d)
+git clone -b record-server-ca-during-registration --depth 1 https://github.com/lcarva/pipeline-service/ $PIPELINE_SERVICE_DIR
+export WORK_DIR="${PIPELINE_SERVICE_DIR}/gitops/sre/"
+export WORKSPACE_DIR=$WORK_DIR
+
+if [ "$ROOT_WORKSPACE" == "~" ]; then
+  ROOT_WORKSPACE=$(KUBECONFIG=${KCP_KUBECONFIG} kubectl ws '~' | cut -f2 -d'"')
+fi
+
+KUBECONFIG=$KCP_KUBECONFIG $PIPELINE_SERVICE_DIR/images/access-setup/content/bin/setup_kcp.sh --kcp-workspace pipeline-service --kcp-org $ROOT_WORKSPACE
+
+KUBECONFIG=$CLUSTER_KUBECONFIG $PIPELINE_SERVICE_DIR/images/access-setup/content/bin/setup_compute.sh
+
+$PIPELINE_SERVICE_DIR/images/cluster-setup/bin/install.sh
+
+$PIPELINE_SERVICE_DIR/images/kcp-registrar/register.sh --kcp-org $ROOT_WORKSPACE --kcp-workspace pipeline-service --kcp-sync-tag v0.8.2
```

Comment:
Need to use KUBECONFIG=$CLUSTER_KUBECONFIG without which it will point to whichever kubeconfig the user currently has exported.

Reply:
the script itself handles this, there is always `KUBECONFIG` variable set before calling `kubectl`. The script is written to work on multiple clusters, it reads kubeconfigs from credentials folder

Reply:
Okay. But how is it different for Line 20 where you're explicitly using KUBECONFIG=$CLUSTER_KUBECONFIG?

Reply:
setup_compute.sh uses `kubectl` command directly and expects that KUBECONFIG is pointing to cluster.

register.sh does not expect that user passes KUBECONFIG since it is taken from credential files.

Reply:
Got it. Thanks for clarifying this.